docs/swagger-ui-bundle/src/test/scala/sttp/tapir/swagger/bundle/SwaggerInterpreterTest.scala

```diff
@@ -0,0 +1,77 @@
+package sttp.tapir.swagger.bundle
+
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import io.circe.generic.auto._
+import org.http4s.HttpRoutes
+import org.http4s.blaze.server.BlazeServerBuilder
+import org.http4s.server.Router
+import org.scalatest.Assertion
+import org.scalatest.funsuite.AsyncFunSuite
+import org.scalatest.matchers.should.Matchers
+import sttp.client3._
+import sttp.model.{StatusCode, Uri}
+import sttp.tapir.generic.auto._
+import sttp.tapir.json.circe.jsonBody
+import sttp.tapir.server.http4s.Http4sServerInterpreter
+import sttp.tapir._
+
+class SwaggerInterpreterTest extends AsyncFunSuite with Matchers {
+
+  case class Author(name: String)
+  case class Book(title: String, year: Int, author: Author)
+
+  val booksListing: PublicEndpoint[Unit, Unit, Vector[Book], Any] = endpoint.get
```

Comment:
we can probably use simpler endpotins (even empty), just to keep the test as simple as possible :)